### ISSUE TITLE
A trailing comma is not permitted after the rest element: const {shouldUpdated, ...props, } = Props;

### ISSUE THREAD

**Issue by mozibrand** [happypancake/react-native-scrollable-tab-view#937]
error: bundling failed: SyntaxError: /Users/superman/Projects/exbuzz-app-react-native/node_modules/react-native-scrollable-tab-view/SceneComponent.js: A trailing comma is not permitted after the rest element (9:32)

   7 | 
   8 | const SceneComponent = (Props) => {
>  9 |   const {shouldUpdated, ...props, } = Props;
     |                                 ^
  10 |   return <View {...props}>
  11 |       <StaticContainer shouldUpdate={shouldUpdated}>

**Comment by Maple13** [comment]
#901

**Comment by Mcl-123** [comment]
I meet the same mistake and don't know how to solve it.

**Comment by mozibrand** [comment]
I solved this problem, I'm not going to wait for the author, I used this branch
https://www.npmjs.com/package/react-native-scrollable-tab-view2

**Comment by ihor** [comment]
@mozibrand could you please create a PR?

**Comment by ihor** [comment]
Actually, this issue is already fixed in the master branch. You can switch to it in your dependencies:
```
"react-native-scrollable-tab-view": "git+https://github.com/happypancake/react-native-scrollable-tab-view.git",
```

**Comment by pvinis** [comment]
changing something in node_modules is not a solution. it's a temporary fix.

**Comment by Clearives** [comment]
A trailing comma is not permitted after the rest element,hope to fix,thanks~

![image](https://user-images.githubusercontent.com/8925862/46988682-5505a680-d12c-11e8-9147-1dce31850e38.png)

**Comment by ptomasroos** [comment]
This has now been published, truly sorry for the delay on this: 0.9.0
